data/remotes.d/README

```diff
@@ -0,0 +1,5 @@
+Steps to make vendor firmware work:
```

Comment:
It would be better to make this README.md (and use markdown format) so that it makes it more readable in the Github web client.